jstz_cli/src/sandbox/daemon.rs

```diff
@@ -173,12 +174,55 @@ fn start_rollup_node(cfg: &Config) -> Result<Child> {
     )
 }
 
-fn set_ticketer(cfg: &Config, bridge_address: &str) -> Result<()> {
-    OctezClient::send_rollup_external_message(
-        cfg,
-        "bootstrap2",
-        &format!("{{ \"SetTicketer\": \"{}\" }}", bridge_address),
-    )
+fn smart_rollup_installer(cfg: &Config, bridge_address: &str) -> Result<()> {
+    //Convert address
+    let address_encoding =
+        serialize(&PublicKeyHash::from_base58(&bridge_address).unwrap());
+    let hex_address = hex::encode(address_encoding.clone());
+
+    let instructions = YamlConfig {
+        instructions: vec![Instr::Set(SetArgs {
+            value: hex_address.clone(),
+            to: "/ticketer".to_owned(),
+        })],
+    };
+    let yaml_config = serde_yaml::to_string(&instructions).unwrap();
+
+    // Define the temporary setup-file with serialised representation of the address computed by octez-codec
+    let ticketer_path = cfg.jstz_path.join("ticketer");
+    fs::create_dir_all(ticketer_path.clone())?;
+    fs::write(ticketer_path.clone().join("setup-file"), yaml_config)?;
```

Comment:
Please create a `/tmp` file instead, this directory + file clutters the `jstz` directory